src/lasp_dependence_dag.erl

```diff
@@ -11,7 +11,9 @@
          add_vertices/1]).
 
 %% Test
+%% @todo Only export on test.
```

Comment:
These functions are useful to check that the dag is correct, but I don't think they should be part of the public api. I'd surround the export with an `ifdef(TEST)` once I set up some test suite for this module.